lib/PersistentCache.jsm

```diff
@@ -59,9 +59,8 @@ this.PersistentCache = class PersistentCache {
         const filepath = OS.Path.join(OS.Constants.Path.localProfileDir, this._filename);
         const fileExists = await OS.File.exists(filepath);
```

Comment:
Out of curiosity, is it still necessary to do the stat here? Can we do the fetch, and handle the failure status in the event that the file does not exist?

Reply:
good point, will fix 👍